src/conversion.ts

```diff
@@ -70,6 +82,77 @@ export function ftom(frequency: number): [number, number] {
   return [midiNoteNumber, centsOffset];
 }
 
+/**
+ * Convert MTS pitch value to 3-byte representation
+ * @param number MTS pitch value
+ * @returns Uint8Array 3-byte of 7-bit MTS data
+ */
+export function mtsToMtsBytes(mtsValue: number): Uint8Array {
+  const noteNumber = Math.trunc(mtsValue);
```

Comment:
Is there a test where `mtsValue` is negative? I'm wondering if we should worry about `Math.trunc` rounding towards zero instead of negative infinity. SW can produce silly scales with nanohertz frequencies.

Reply:
Fair call, I think that would only happen with bad usage but it would be good practice to clamp it to 0 otherwise you could be producing junk byte data.